certora/specs/ERC20Invariants.spec

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+methods {
+    function totalSupply() external returns uint256 envfree;
+    function balanceOf(address) external returns uint256 envfree;
+    function delegatee(address) external returns address   envfree;
+    function delegatedVotingPower(address) external returns uint256   envfree;
+    function upgradeToAndCall(address, bytes)      external => NONDET DELETE;
```

Comment:
```suggestion
    function delegatee(address) external returns address envfree;
    function delegatedVotingPower(address) external returns uint256 envfree;
    function upgradeToAndCall(address, bytes) external => NONDET DELETE;
```

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/102/commits/9de63d8114da2edb1983cd41928262cc46be5dcc.